src/AppInstallerCLICore/Commands/DscCommand.cpp

```diff
@@ -12,8 +12,22 @@
 
 namespace AppInstaller::CLI
 {
+    namespace
+    {
+        Argument GetOutputFileArgument()
+        {
+            return { Execution::Args::Type::OutputFile, Resource::String::OutputFileArgumentDescription, ArgumentType::Standard };

```

Comment:
nit: Do we consider differentiating output file vs output directory?

Reply:
Changed the description, left the arg type the same.